CommonUI/Sources/CommonUI/MovieUI/MovieDetails/MovieDetailsView.swift

```diff
@@ -2,13 +2,13 @@ import SwiftUI
 import ApiService
 import UserData
 
-public struct MovieDetails: View {
-    let movie: Movie
-    @State private var isFavorite: Bool
+public struct MovieDetailsView: View {
+    @ObservedObject private var viewModel: MovieDetailsViewModel
+    private let movie: Movie
```

Comment:
this var is not needed. since this `MovieDetailsView` references the `movie` through the `viewModel` like `viewModel.movie.title`

Reply:
removed it.